backend/danswer/configs/model_configs.py

```diff
@@ -61,6 +61,8 @@
 GEN_AI_MODEL_PROVIDER = os.environ.get("GEN_AI_MODEL_PROVIDER") or "openai"
 # If using Azure, it's the engine name, for example: Danswer
 GEN_AI_MODEL_VERSION = os.environ.get("GEN_AI_MODEL_VERSION")
+# The fallback display name to use for default model when using a custom model provider
+GEN_AI_DISPLAY_NAME = os.environ.get("GEN_AI_DISPLAY_NAME") or "Custom LLM"
```

Comment:
Why not just use `os.environ.get("GEN_AI_DISPLAY_NAME", "Custom LLM")`?

Reply:
I was trying to follow the pattern used everywhere else in this file 🙂